### ISSUE TITLE
Renaming notes in File Explorer/Sidebar cuts note name off

### ISSUE THREAD

**Issue by jloh** [kepano/obsidian-minimal#568]
### Is the bug present when using the default Obsidian theme?

No

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.3.3

### Describe the bug

The same thing that happened in https://github.com/kepano/obsidian-minimal/issues/272 is now happening again. When trying to rename notes the whole note name gets cut off in the sidebar so it becomes impossible to do so.

### Debug info

```
SYSTEM INFO:
	Obsidian version: v1.3.1
	Installer version: v1.1.9
	Operating system: Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:28 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6000 22.4.0
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: Minimal v6.3.3
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 21
	Plugins enabled: 17
		1: Advanced URI v1.35.0
		2: Better Command Palette v0.15.0
		3: Buttons v0.4.19
		4: Calendar v1.5.10
		5: Contextual Typography v2.2.5
		6: Dataview v0.5.55
		7: Default New Tab Page v0.11.0
		8: Excalidraw v1.9.1
		9: Influx v2.1.2
		10: Minimal Theme Settings v6.3.1
		11: Paste URL into selection v1.7.0
		12: Periodic Notes v0.0.17
		13: Quick Switcher++ v3.3.0
		14: QuickAdd v0.23.0
		15: Style Settings v1.0.3
		16: Tasks v3.7.0
		17: Templater v1.16.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by jloh** [comment]
Disabling line trimming in the sidebar via Minimal Theme Settings does fix it

**Comment by jloh** [comment]
I can still replicate this in the latest release. Same thing, turning off line trimming fixes it.

**Comment by kepano** [comment]
Is this present in the default theme?

**Comment by kepano** [comment]
Fixed in 6.3.7

**Comment by jloh** [comment]
Thankyou! Can confirm this is fixed.
